app/infrastructure/queues/supabase_queue.py

```
@@ -363,97 +371,183 @@ async def complete_job(self, job_data: Dict[str, Any], job_tracer:Optional[JobTr
-
-            # Delete current message and re-queue with delay
-            await self.queue.delete(queue_name, msg_id)
-            _ = await self.queue.send(queue=queue_name, message=retry_job_data, delay=retry_delay)
-
-            logger.info(
-                f"Job {job_data.get('id')} scheduled for retry {attempts}/{max_attempts} in {retry_delay}s"
-            )
-            
-            return perma_failure, True
+        )
+        # Strip queue-generated IDs; we’re re-inserting a fresh message
```

Ruff (0.13.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)